### ISSUE TITLE
Obj-C support is broken

### ISSUE THREAD

**Issue by joeljfischer** [Quick/Nimble#642]
- [x] I have read [CONTRIBUTING](https://github.com/Quick/Nimble/blob/master/CONTRIBUTING.md) and have done my best to follow them.

### What did you do?
Updated Nimble version from 7.3.0 to 8.0.1.

### What did you expect to happen?
Tests still build

### What actually happened instead?
My unit tests using Quick & Nimble will not compile:

```
Property 'to' cannot be found in forward class object 'NMBExpectation'
```

This points to `DSL.h` which has `@class NMBExpectation`.

### Environment

List the software versions you're using:

 - Quick: *2.0.0*
 - Nimble: *8.0.1*
 - Xcode Version: *10.2 (10E125)* (Open Xcode; In menubar: Xcode > About Xcode)
 - Swift Version: *Objective-C* (Open Xcode Preferences; Components > Toolchains. If none, use `Xcode Default`.)

Please also mention which package manager you used and its version. Delete the
other package managers in this list:

 - Carthage: *0.32.0* (Use `carthage version` in Terminal)

**Comment by joeljfischer** [comment]
This seems like it may be an issue with Xcode 10.2? I reverted to 7.3.4 and I'm having the same issue.

**Comment by joeljfischer** [comment]
This appears to be related to `Nimble-Swift.h` which actually defines `NMBExpectation` not being available in `Nimble.h`. However, I'm still unable to get it to compile and expose the information in such a way that my tests work properly. I'm still looking into it, but if someone with more familiarity with Nimble's internals could take a look as well, it would be greatly appreciated.

**Comment by ikesyo** [comment]
I don't see any problem for Objective-C support, e.g. https://github.com/ReactiveCocoa/ReactiveObjC/pull/156

**Comment by joeljfischer** [comment]
I did also try that, and it still failed. I tested with a submodule and everything worked fine, so I suspected Carthage was the issue. It appears I was correct and my issue was resolved in [v0.33.0](https://github.com/Carthage/Carthage/releases/tag/0.33.0) which released late yesterday. After updating, everything worked properly again.
